cert/openssl.cnf

```diff
@@ -135,260 +140,366 @@ caIssuers;URI.0 = http://cacerts.digicert.com/WinnForumCA.crt
 OCSP;URI.0 = http://ocsp.digicert.com
 
 ####################################################################
-[ wif_root_ca ]
+[ root_ca ]
 
 subjectKeyIdentifier	= hash
 authorityKeyIdentifier	= keyid:always,issuer
 basicConstraints 		= critical, CA:true
 keyUsage 				= critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA
+
+[ cps ]
 
-ROLE = ASN1:UTF8String:CA
+policyIdentifier = 2.16.840.1.114412.2.1
+ CPS.1="https://www.digicert.com/CPS"
 
 ####################################################################
-[ wif_sas_ca ]
+[ sas_ca ]
 
 subjectKeyIdentifier = hash
 #authorityKeyIdentifier = keyid:always,issuer
 basicConstraints = critical, CA:true, pathlen:0
 keyUsage = critical, digitalSignature, cRLSign, keyCertSign
+certificatePolicies=@cps,ROLE_CA,ROLE_SAS
 
-ROLE=ASN1:SEQUENCE:sas_seq
+[ cps ]
```

Comment:
Right now the different [cps] sections are superfluous, but if different CAs operated different parts of the infrastructure, each profile may have its own CPS/CP which applies to it so I put in a [cps] section for each profile so they could be edited independently in the future.